amt/wsman.py

```diff
@@ -36,12 +38,13 @@
     'cd': 'Intel(r) AMT: Force CD/DVD Boot',
 }
 
-
+FRIENDLY_POWER_STATE = {v: k for (k, v) in six.iteritems(POWER_STATES)}
 
 def friendly_power_state(state):
-    for k, v in POWER_STATES.items():
-        if v == int(state):
-            return k
+    try:
```

Comment:
I think .get(..., 'unknown') is probably really what we want here, as this is for display only.